crates/optimism/src/evm.rs

```diff
@@ -283,7 +283,6 @@ mod tests {
     }
 
     #[test]
-    #[cfg(feature = "blst")]
     fn test_halted_tx_call_bls12_381_g1_add_out_of_gas() {
         let ctx = Context::op()
             .modify_tx_chained(|tx| {
```

Comment:
Since both arkworks and blst are optional, these would need to be added back, since we test with no-default-features and no feature flags

Reply:
Done